docs/plugins/fetchart.rst

```diff
@@ -90,6 +90,10 @@ file. The available options are:
   instructed to store cover art as non-progressive JPEG. You might need this if
   you use DAPs that don't support progressive images.
   Default: ``no``.
+- **cover_format**: If enabled, forced the cover image into the specified
+  format. Valid values line up with ImageMagick/Pillow decoder names (for
+  instance, ``PNG`` or ``JPEG``). Also respects ``deinterlace``.
```

Comment:
Maybe we should just give a list of all the formats we expect to work? We could just start with `PNG` and `JPEG`, for example, and not bother people with the somewhat subtle advice to look in the ImageMagick/Pillow documentation. (And if folks want other formats, maybe we can test those and add them to the docs.)

Reply:
This might be a bit more complex. These two are the ones I tested and my confidence isn't high that the two libraries use matching names for others. I'll look for the lists in a bit.

Reply:
The list can be obtained via `magick identify -list format` and via `PIL.features.pilinfo()` for Pillow.

Reply:
Yeah, I think that's the practical thing—just listing a few common formats that 99.99% of people will want to use seems like the most helpful thing. And yes, supplementing that with a docs link will help satisfy the curiosity of the remaining 0.01%.

Reply:
Got it, I'll change that and squash the commits.